plugin/trino-redshift/pom.xml

```diff
@@ -23,12 +23,22 @@
             <artifactId>trino-base-jdbc</artifactId>
```

Comment:
This change is backward incompatible change. It is changing semantics of Redshift connector. One could say that after this change it is a new different Redshift connector. Most of the existing production usages of Redshift will now require a migration to make Trino upgrade. Such migration is a binary step. Either user is adjusted to use 403 Redshift connector or 404. There is no middle ground, so rolling back such migration is also difficult.

Reply:
Let's restore old one as `redshift-legacy`?

Reply:
I am not sure if it is enough. I would prefer to rename the existing one to mark it is deprecated (legacy) and create a new one with unique name (`redshift-v2`?). Then after some time once legacy is removed I would rename the one to `redshift`)